HISTORY.rst

```diff
@@ -5,6 +5,16 @@ History
 .. towncrier release notes start
 
 
+Parsec v2.14.1-rc2 (2022-11-23)
```

Comment:
Aren't we're missing the change from the `updated message display to the user during invitation` ?